incognia.go

```diff
@@ -98,6 +98,18 @@ func New(config *IncogniaClientConfig) (*Client, error) {
 	return client, nil
 }
 
+func (c *Client) Authenticate() error {
+	return c.tokenManager.refreshToken()
+}
+
+func (c *Client) GetAuthenticationExpiresInSeconds() int64 {
```

Comment:
Instead of exposing `token.ExpiresIn`, that would imply we'll always use a token with that information available. I would use `token.CreatedAt` and `token.ExpiresIn` to calculate a new value `ExpiresAt` and have it returned to the user as `time.Time`.